spock-specs/src/test/groovy/org/spockframework/smoke/AnonymousInnerClasses.groovy

```diff
@@ -19,7 +19,7 @@ package org.spockframework.smoke
 import org.spockframework.EmbeddedSpecification
 
 class AnonymousInnerClasses extends EmbeddedSpecification {
-  def "anonymous inner class and dot in feature name does not fail execution"() {
+  def "anonymous inner class and dot in feature name does not fail execution (Groovy 2)"() {
```

Comment:
The names still have the added Groovy 2. ;-)

Reply:
Nice catch!